.github/workflows/main.yml

```diff
@@ -8,7 +8,7 @@ env:
 
 jobs:
   build:
-    name: Continuous integration
+    name: Build
```

Comment:
How about "CI"? It would be a bit more precise.

Reply:
I found that we should put a build configuration name like "Build with Go 1.11", etc. here. I put "Continuous integration" as a name of a workflow, which is upper layer of a job.